### DISCUSSION TITLE
policy exclusions for specific repos

### DISCUSSION THREAD

**Discussion by OneCyrus** [community/community#35185] [Codespaces]
currently the codespaces policy only allows to set policy on specific repos or on all. but our use case is mostly in the other direction. we would like to limit all repos per default to small 2 core VMs and authenticated port forwarding. every new repo should automatically get this policy applied. this works currently with this specific policy applied to all repos. 
BUT there are some repos which should be allowed to use larger VMs and less restrictions in general. So I thought we could add new policies to exclude those repos from those restrictions, but additional policies don't allow to remove restrictions as they are cumulative only.

there should be a way to set policies as overrides/exclusions to solve this problem.

**Comment by tanmayeekamath** [comment]
Hey @OneCyrus thanks for your feedback. 
We have plans to add support for policy violations & exceptions. Violations will help you as as admin get a list of resources that are out of compliance as a result of a policy being applied/updated whereas exceptions will enable you grant a different set of resources for one-off repos. I'll plus one this work-item in our backlog with your feedback. 
We don't yet have a timeline to add support for this, but you can subscribe to our [public roadmap](https://aka.ms/ghcs-roadmap) where we'll update you on timelines as soon as we have them. Thanks for your patience.

**Reply by OneCyrus** [reply]
Hi @tanmayeekamath, that sounds good. Thanks
